index.js

```diff
@@ -2,10 +2,17 @@ const core = require("@actions/core");
 const exec = require("@actions/exec");
 const { promises: fs } = require("fs");
 
-const host =
-  core.getInput("owner") === "Staging-ObelusFamily"
-    ? "https://engine-staging.wilco.gg"
-    : "https://engine.wilco.gg";
+const host = () => {
+   if (core.getInput("owner") === "Staging-ObelusFamily") {
+     const company = core.getInput("repo").split("-")[0]; 
+     if (company === "Anythink") {
+       return "https://engine-staging.wilco.gg";
+     } else {
+       return `https://${company}.ngrok.io`;
```

Comment:
what does `company` equal to locally? I'm confused. Could you describe in the PR description how it's used? do I have to start ngrok with a custom domain based on my repo name?